### PR TITLE
111920 fix snooty errors

### PR DESCRIPTION
**Pull request by ccho-mongodb**
### Issue JIRA link:
None

### Docs staging link (requires sign-in on MongoDB Corp SSO):
Error-free workerpool log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5fb68ab305418b8e43343c0b

### CODE REVIEW

File: source/fundamentals/data-formats/codecs.txt [hunk 1]
```diff
@@ -46,3 +46,30 @@ Custom Codec Example
 - Sample query to demonstrate decode results (of a field in a POJO) using custom codec
 - Sample insert and resulting fields of the document to demonstrate encode 
 - Example showing how to read/encode a document with unknown fields
+
```
Comment (ccho-mongodb) [inline source/fundamentals/data-formats/codecs.txt:49]: Accidental commit of some notes -- there's a ticket to create the content for this page in another PR.

Comment (terakilobyte) [reply source/fundamentals/data-formats/codecs.txt:49]: So is this PR contingent on that other PR?

Comment (ccho-mongodb) [reply source/fundamentals/data-formats/codecs.txt:49]: They're just notes that will get replaced by actual content in that PR.

**Comment by terakilobyte** [comment]
🎆

**Review by terakilobyte — APPROVED** [review]
LGTM, love seeing the error-free log ✨

**Pull request merged**